processing/thermal.py

```diff
@@ -60,30 +60,18 @@ def process(recording, conf):
             classify(conf, recording, api, s3, logger)
 
 
-def classify_models(api, command, conf):
-    """ classifies all models described in the config """
-
-    model_results = []
-    for model in conf.models:
-        model_result = classify_file(api, command, conf, model)
-        model_results.append(model_result)
-
-    return model_results
-
-
-def classify_file(api, command, conf, model):
-
-    command = "{} -m {} -p {}".format(command, model.model_file, model.preview)
+def classify_file(api, command, conf):
+    command = "{}".format(command)
     classify_info = run_classify_command(command, conf.classify_dir)
 
-    track_info = classify_info["tracks"]
-    formatted_tracks = format_track_data(track_info)
+    format_track_data(classify_info["tracks"])
 
     # Auto tag the video
-    tagged_tracks, tags = calculate_tags(formatted_tracks, conf)
+    algorithm_id = api.get_algorithm_id(classify_info["algorithm"])
```

Comment:
The algorithm id should load information specific to each algorithm.   So if there are multiple models that are being processed I thought you would need mulitple algorithm ids for each one.   I guess they could all use the same one, (as long as it is obvious from the names what applies to what). but it's not how it was intended.

Reply:
I think there is no need as the only thing different in each is model name and model file which is being saved in the data, so algorithm_id i am using to represent the tracking algorithm. I think this makes more sense

Reply:
Especially since it applies to the tracks